packages/common/components/nodes/youtube-card.marko

```diff
@@ -0,0 +1,37 @@
+import { getAsObject, get } from "@base-cms/object-path";
+
+$ const video = getAsObject(input, "node");
+$ const primaryImage = getAsObject(video, "primaryImage");
+$ const withDescription = input.withDescription != null ? input.withDescription : true;
+
+$ const { linkAttrs } = input;
+
+<marko-web-node
+  type="video-content"
```

Comment:
This isn't used. Remove

Reply:
The `--video-content-type` modifier appends the video play icon overlaya